cmd/pbm/delete.go

```diff
@@ -44,6 +46,15 @@ func deleteBackup(
 	if d.bcpType != "" && d.olderThan == "" {
 		return nil, errors.New("cannot use --type without --older-than")
 	}
+	if d.profile != "" {
```

Comment:
Probably we should forbidden delete by name in combination with `--profile`:
```
pbm delete <backup-name> --profile p1
```
In case when backup-name is on default storage or on e.g. `p2`, above command will still delete backup, so that's kind of misleading.

Alternative is to not delete that backup in case when that name is not present within the profile, but I'd recommend the other one.